docs/docsite/rst/community/committer_guidelines.rst

```diff
@@ -4,50 +4,58 @@
 Committers Guidelines
 *********************
 
-These are the guidelines for people with commit privileges on the Ansible GitHub repository. Committers are essentially acting as members of the Ansible Core team, although not necessarily as employees of Ansible and Red Hat. Please read the guidelines before you commit.
+These are the guidelines for people with commit privileges on the Ansible GitHub repositories.
 
-These guidelines apply to everyone. At the same time, this ISN'T a process document. So just use good judgment. You've been given commit access because we trust your judgment.
+Committers of `Ansible-core <https://github.com/ansible/ansible>`_ are necessarily Red Hat employees acting as members of the Ansible Core team. Committers of `Ansible collections <https://github.com/ansible-collections/>`_ can be Ansible Core team members as well as independent community members (collection maintainers). Please read the guidelines before you commit.
```

Comment:
Do we really not have any community committers to the ansible/ansible repo?

Reply:
as far as i know, no, we don't. @gundalow ^ what do you think?

Reply:
They've all been removed.

Reply:
@gundalow thanks for confirming!